code/modules/admin/admin_verbs.dm

```diff
@@ -659,6 +659,9 @@ var/list/admin_verbs_hideable = list(
 
 	if(!holder)
 		return
+	
+	if(has_antag_hud())
+		toggle_antag_hud(TRUE)
```

Comment:
this looks bad.
typically with a ```toggle_X()``` proc they have an ```override``` or ```force``` argument, that forces it into that state (eg: the proc defaults to toggling but can also be used for forcing on or off), except in this case you're turning the hud off!

Off? True? WHO DO I BELIEVE!?